docs/components/connectors/out-of-the-box-connectors/google-maps-platform.md

```diff
@@ -0,0 +1,106 @@
+
+### Get place ID
+
+This operation allows you to get the Google Maps place ID by address.
+To use this operation, enable the [Google Places API](https://developers.google.com/maps/documentation/places/web-service). See the [official documentation](https://developers.google.com/maps/documentation/places/web-service/get-api-key) for more information on enabling the Google API.
+
+### Calculate distance
+
+This operation allows you to calculate a distance between two place IDs.
+To use this operation, enable the [Google Directions API](https://developers.google.com/maps/documentation/directions). See the [official documentation](https://developers.google.com/maps/documentation/directions/get-api-key) for more information on enabling the Google API.
+
+## Usage
+
+### Address validation, formatting, getting postal address
+
+1. Select **Validate Address** from the **Operation type** dropdown in the **Operation** section.
+2. Populate the **Authentication** section as described in the [respective section](#authentication).
+3. (Optional) In the **Input** section, set **Region Code** (i.e `US`). You can find supported region codes [here](https://developers.google.com/maps/documentation/address-validation/coverage).
+4. (Optional) In the **Input** section, set **Locality**, an incorporated city or town political entity (i.e `Mountain View`).
+5. In the **Input** section, set **Address**, an incorporated city or town political entity (i.e `1600 Amphitheatre Pkwy`).
+6. In the **Output** section set **Result Variable** or **Result Expression**. See the [response mapping documentation](/docs/components/connectors/use-connectors.md#response-mapping) to learn more.
+7. Get from response needed information. Find a full example of the response [here](https://developers.google.com/maps/documentation/address-validation/requests-validate-address#address_validation_response). For example, if you want to get postal address and formatted address, you can set to **Result Expression** next FEEl Expression :
+
+```
+{
+ formattedAddress: response.body.result.address.formattedAddress,
+ postalAddress: response.body.result.address.postalAddress
+}
+```
+
+### Get place ID
+
+1. Select **Get Place ID** from the **Operation type** dropdown in the **Operation** section.
+2. Populate the **Authentication** section as described in the [respective section](#authentication).
+3. In the **Input** section, set **Address**. This address can be `formatedAddress`, which you can get using [this example](#address-validation-formatting-getting-postal-address).
+4. In the **Output** section in the **Result Expression** property, you can change the name of the `placeId` response. By default, in **Result Expression** properties you can find next expression:
```

Comment:
"By default, in **Result Expression** properties you can find next expression:" -- what do you mean here?

Reply:
Tweaked in my latest commit -- does this make sense?